.github/workflows/cran-check.yaml

```diff
@@ -0,0 +1,125 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on: push
+
+name: cran-check
+
+jobs:
+  check_version_number:
+    runs-on: ubuntu-latest
+    name: Check version numbers of packages
+    outputs:
+        matrix: ${{ steps.get_packages.outputs.value }}
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+      R_KEEP_PKG_SOURCE: yes
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: r-lib/actions/setup-pandoc@v2
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - id: get_packages
+        run: |
+          install.packages('data.table')
+          library('data.table')
+
+          # Read files in drat folder, extract names, and filter to max versions of package found.
+          dratPackages = data.table(filename = list.files(file.path('src', 'contrib')))
+          dratPackages[, filename := gsub('.tar.gz', '', filename, fixed = TRUE)]
+          dratPackages = dratPackages[, c('name', 'version') := tstrsplit(filename, '_', fixed = TRUE)]
+          dratPackages = dratPackages[
+            !is.na(version), .(drat_version = max(version)), by = name]
+
+          # Establish version levels and split out into separate columns
+          versionLevels = c('major', 'minor', 'patch', 'dev')
+          dratPackages[, (paste0('drat_', versionLevels)) :=
+                        tstrsplit(drat_version, '.', fixed = TRUE, type.convert = TRUE)]
```

Comment:
This line should start only two spaces past the previous, similar to lines 34 and 35.